packages/react/src/components/Table/EmptyTable/EmptyTable.jsx

```diff
@@ -102,7 +69,36 @@ const EmptyTable = ({
             emptyState
           ) : (
             <div className="empty-table-cell--default">
-              {isFiltered ? isFilterEmptyTable : defaultEmptyTable}
+              {isFiltered ? (
+                <EmptyState
+                  icon="no-result"
+                  title={messageWithFilters}
+                  body={messageWithFiltersBody || ''}
+                  action={
+                    onEmptyStateAction
+                      ? {
+                          label: buttonLabelWithFilters,
+                          onClick: onEmptyStateAction,
+                          kind: 'secondary',
+                        }
+                      : null
+                  }
+                />
+              ) : (
+                <EmptyState
+                  icon={emptyStateIcon ?? 'empty'}
```

Comment:
This is breaking monitor code. The change should not break downstream application.
emptyStateIcon is by default empty or '' so '' ?? 'empty' always evaluated to ''

Reply:
If User will pass props for empty icon then it will take that else it will take 'empty' icon.
we have below possible option for emptyStateIcon.

const icons = {
  error: ErrorImage,
  error404: Error404Image,
  empty: EmptyImage,
  'not-authorized': NotAuthImage,
  'no-result': NoResultImage,
  success: SuccessImage,
};